RUMM-1603: Add suggestion to disable errors and resources tracking while in DEV mode (#104)
DataDog/dd-sdk-reactnative: README.md

* `First Byte` - The time between the scheduled request and the first byte of the response. This includes time for the request preparation on the native level, network latency, and the time it took the server to prepare the response.
* `Download` - The time it took to receive a response.

## Development mode

> While in a development mode, your application can submit extra events related to the React Native tooling: code transformation errors, requests to a local development server, etc.

Reviewer: ```suggestion
While in development mode, your application can submit extra events related to the React Native tooling, like code transformation errors, requests to a local development server, etc.
```

> To prevent these event from showing in the dashboard, we recommend you to disable errors and resources tracking in the development mode, you can use `__DEV__` flag for that:

Reviewer: ```suggestion
To prevent these event from showing in the dashboard, Datadog recommends you to disable errors and resources tracking in dev mode, using the `__DEV__` flag:
```

Reviewer 2: I'm not sure we should "recommend" disabling errors and network. We should never recommend people to disable our features. I'd rather say 
> "To prevent these event from showing in the dashboard, you can disable errors and resources tracking in dev mode, using the `__DEV__` flag:"
